src/App.tsx

```diff
@@ -65,12 +68,9 @@ const App: React.FC = () => {
 							element={
 								isLoggedIn ? (
```

Comment:
Let's not check every component this way now it's ok but with app growing that won't be optimal to check every component with this value. You can change the architecture of having some parent to show if logged in and inside of it distribute children. Or some other way you can come up with. Can be discussed